README.md

```diff
@@ -1,6 +1,6 @@
 # netCLI
 
-![Logo](inausoft.netCLI.all/inausoft.netCLI/assets/netCLI.png)
+![Logo](src/inausoft.netCLI/assets/netCLI.png)
```

Comment:
I would consider to rename solution from `inausoft.netCLI.all` into simple `inausoft.netCLI`

Reply:
Agreed.